localize.js

```diff
@@ -3,16 +3,17 @@ const Packet = require('./packet.js');
 const LoginSuccess = require('./packets/clientbound/loginSuccess.js');
 const log = require('./utility.js').log
 
-function localizePacket(packetData, x, z) {
-  var packet = Packet.loadFromBuffer(packetData)[0]
-  var packetType = identify(packet, 0)
+function localizePacket(packet, x, z, ctx) {
```

Comment:
Changed the input of this method from being data to being a packet. Makes more sense for when we use this after parsing the packets.